cpp/benchmarks/fixture/benchmark_fixtures.hpp

```diff
@@ -0,0 +1,56 @@
+#include <benchmark/benchmark.h>
```

Comment:
You should include rmm headers. It works in this PR because one of the headers included before this in apply_boolean_benchmark.cu already includes it.

Reply:
Done, thanks.